<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pi-Karaoke</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="TimeSlider">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>610</y>
      <width>860</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>860</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>860</width>
      <height>20</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>631</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ReplayButton">
       <property name="text">
        <string>Replay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlayButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MuteButton">
       <property name="text">
        <string>Mute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="VolumeSlider">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>660</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="EndTime">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>610</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="Volume">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>660</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="VolumeLable">
    <property name="geometry">
     <rect>
      <x>985</x>
      <y>660</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Volume:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Playlist">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>225</width>
      <height>645</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>225</width>
      <height>645</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Song Name :</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LyricDisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>580</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="LyricNext">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>555</y>
      <width>340</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>340</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>340</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlaylistLable">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Playlist:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchBar">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>15</y>
      <width>795</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="FieldSelection">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>15</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>By Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By Singer</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="SearchLable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>15</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Search:</string>
    </property>
   </widget>
   <widget class="QLabel" name="SongInfo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>1010</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="BrowseList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>55</y>
      <width>1000</width>
      <height>410</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <kerning>true</kerning>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Song Name :　　　　　　　　　　Singer :</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
